### PR TITLE
Bug 1884150: Fix timeout issue in ceph test suite

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

**Review by shyRozen — APPROVED** [review]

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts [hunk 1]
```diff
@@ -28,9 +28,9 @@ describe('Check health data on main OCP dashboard ', () => {
     await browser.sleep(2 * SECOND);
     expect(mainHealtYellowSvg.isPresent()).toBe(true);
     noOutChange('unset');
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 300;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL += 30 * SECOND;
     await browser.wait(until.presenceOf(mainHealtGreenSvg));
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts:32]:
```suggestion
    await browser.wait(until.presenceOf(mainHealtGreenSvg), 60 * SECOND);
```

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts [hunk 2]
```diff
@@ -28,9 +28,9 @@ describe('Check health data on main OCP dashboard ', () => {
     await browser.sleep(2 * SECOND);
     expect(mainHealtYellowSvg.isPresent()).toBe(true);
     noOutChange('unset');
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 300;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL += 30 * SECOND;
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts:31]:
```suggestion
```

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts [hunk 3]
```diff
@@ -28,9 +28,9 @@ describe('Check health data on main OCP dashboard ', () => {
...
     await browser.wait(until.presenceOf(mainHealtGreenSvg));
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL -= 30 * SECOND;
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts:33]:
```suggestion
```

**Review by bipuladh — CHANGES_REQUESTED** [review]

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**